scripts/apps/vocabularies/controllers/VocabularyConfigController.ts

```diff
@@ -60,6 +62,30 @@ export function VocabularyConfigController($scope: IScope, $route, $routeParams,
         $route.updateParams({id: vocabulary._id});
     };
 
+    /**
+     * Download vocabulary in json file.
+     *
+     * @param {Object} vocabulary
+     */
+    $scope.downloadVocabulary = (vocabulary: IVocabulary) => {
+        const _vocabulary = JSON.stringify(vocabulary);
+        const _vocabularyUri = 'data:application/json;charset=utf-8,' + encodeURIComponent(_vocabulary);
```

Comment:
Could you extract file downloading logic to a separate function? Something like [`downloadBlob`](https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/core/helpers/utils.tsx#L97)